src/main/scala/org/constellation/primitives/ConcurrentTipService.scala

```diff
@@ -213,4 +217,43 @@ class ConcurrentTipService[F[_]: Concurrent: Logger: Clock](
       }
     selectedFacils.toMap
   }
+
+  private def filterFacilitators(facilitators: Map[Id, PeerData])(contextToReturn: ContextShift[F]) =
+    for {
+      facilitatorHeights <- getFacilitatorHeights(facilitators)(contextToReturn: ContextShift[F])
```

Comment:
We already have that info while performing health check we could store it there and only query from here then extra api call wouln't be needed and since it's not crucial to have this info up to date I would go for such solution. What you think ? @mchrapek @marcinwadon